docs/running-node.md

```diff
@@ -218,9 +241,9 @@ $ dclcli tx validator add-node \
 
 If the transaction has been successfully written you would find `"success": true` in the output JSON.
 
-#### Notify other validator administrators
+#### Notify other validator admins
 
-Provide the node's `id`, `ip` and peer port (by default `26656`) to other validator administrators
+Provide the node's `id`, `ip` and peer port (by default `26656`) to other validator admins.
```

Comment:
Makes sense to clarify how to get the node's id.

Reply:
updated